pkgs/applications/editors/neovim/tests/default.nix

```
@@ -1,5 +1,5 @@
 # run tests by building `neovim.tests`
-{ vimUtils, vim_configurable, writeText, neovim, vimPlugins
+{ vimUtils, vim-full, writeText, neovim, vimPlugins
```

```suggestion
{ vimUtils, writeText, neovim, vimPlugins
```
If it is unused we might as well remove it